Recipe Detail/recipe.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="FoodApp is the first Application for food delivery for Gaza Restaurants, ... FoodApp is Palestinian foodies favorite food ordering app and website"
+    />
+    <title>Recipe Detail</title>
+    <link rel="stylesheet" href="style.css" />
```

Comment:
Link the css file correctly because there is a mistake in the file name, you can put it
`<link rel="stylesheet" href="../Recipe Detail/recipe.css" />`